api-flask/app/zonal_stats.py

```diff
@@ -161,13 +163,17 @@ def calculate_stats(
         is_path = False
         stats_input = [s.get('geom') for s in zones.get('features')]
 
-    stats = zonal_stats(
-        stats_input,
-        geotiff,
-        stats=stats,
-        prefix=prefix,
-        geojson_out=geojson_out
-    )
+    try:
+        stats = zonal_stats(
+            stats_input,
+            geotiff,
+            stats=stats,
+            prefix=prefix,
+            geojson_out=geojson_out
+        )
+    except Exception as e:
```

Comment:
Similar to the previous comment. We could wrap this exception into a RasterioError one.